src/index.ts

```diff
@@ -1,3 +1,52 @@
 import "./devices/devices-list";
 import "./components/esphome-header-menu";
 import "./components/esphome-fab";
+import "./editor/esphome-editor";
```

Comment:
We should not load the editor in the entrypoint. This means it will load the full editor just to show the cards! Instead, add `import("./editor/esphome-editor");` to firstUpdated()

Reply:
ok, thanks for the hint